xarray/test/test_plot.py

```diff
@@ -2,6 +2,16 @@
 from __future__ import division
 from __future__ import print_function
 
+try:
+    import matplotlib as mpl
+    # Using a different backend makes Travis CI work
+    # This has to happen before other mpl imports
+    mpl.use('Agg')
+    # Order of imports is important here.
+    import matplotlib.pyplot as plt
+except ImportError:
+    pass
```

Comment:
Would you like me to add something here? (note that I just changed this blocks location further up in the module)